baseline.gradle

```
@@ -74,8 +65,7 @@ subprojects {
     spotless {
       java {
         target 'src/main/java/**/*.java', 'src/test/java/**/*.java', 'src/testFixtures/java/**/*.java', 'src/jmh/java/**/*.java', 'src/integration/java/**/*.java'
-        // we use an older version of google-java-format that is compatible with JDK 8
-        googleJavaFormat("1.7")
+        googleJavaFormat()
```

```suggestion
        googleJavaFormat("1.23.0")
```
I would prefer to hardcode this to the latest version